migrations/versions/54e57fb2e1c6_add_contract_source_to_records_and_items.py

```diff
@@ -17,8 +17,8 @@
 
 
 def upgrade():
```

Comment:
let's change the file name to `54e57fb2e1c6_add_publisher_project_source_to_records_and_items.py

Reply:
Ok I can do that